thonny/tktextext.py

```diff
@@ -249,6 +250,23 @@ def dispatch(event):
         if platform.system() == "Windows":
             self.bind("<KeyPress>", self._insert_untypable_characters_on_windows, True)
 
+    def _bind_keypad(self):
+        kmap = {
+            '<KP_Left>': '<Left>',
+            '<KP_Right>': '<Right>',
+            '<KP_Up>': '<Up>',
+            '<KP_Down>': '<Down>',
+            '<KP_Home>': '<Home>',
+            '<KP_End>': '<End>',
+            '<KP_Next>': '<Next>',
+            '<KP_Prior>': '<Prior>',
+            '<KP_Enter>': '<Return>',
+        }
+        for i in kmap:
+            def mfunc(event, key=i):
+                self.event_generate(kmap[key], **{'state': event.state})
+            self.bind(i, mfunc)
```

Comment:
In Windows it gives me `_tkinter.TclError: bad event type or keysym "KP_Left"`. Numpad left keycode is just `Left` in Windows. You should catch and silence the binding errors.

I'm also worried that on some platforms this approach may generate double movement actions (one for KP_Left and another for generated Left). The standard approach is ending the event handler with `return "break"` -- this should stop the event propagation.

Reply:
Also, please add link to the issue as comment inside `bind_keypad`!

Reply:
I created a new pull request rebased and squashed onto current master branch.